app/models/actor.rb

```diff
@@ -0,0 +1,3 @@
+class Actor < ApplicationRecord
+  has_and_belongs_to_many :movies 
```

Comment:
similar note as in the Movie model -- this is not the best association to use when we need the actors_movies to be its own entity